code/game/objects/items/toys.dm

```diff
@@ -1086,3 +1087,182 @@
 	desc = "Small mechanical canary in a cage, does absolutely nothing of any importance!"
 	icon = 'icons/mob/animal.dmi'
 	icon_state = "canary"
+
+/obj/item/toy/balloon
+	name = "balloon"
+	desc = "A simple balloon."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "balloon_deflated"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/toys.dmi', "right_hand" = 'icons/mob/in-hand/right/toys.dmi')
+	w_class = W_CLASS_TINY
+	force = 0
+	throwforce = 0
+	var/col = "#FFFFFF"
+
+/obj/item/toy/balloon/New(atom/A, var/ccol)
```

Comment:
`ccol` is a pretty bad name, `chosen_col` would be better if that's what you're going for.